app/assets/stylesheets/typography.scss

```diff
@@ -4,7 +4,7 @@
 // Alternate base fonts
 body.sans-serif-article-body {
   .body,
-  .crayons-article__body {
```

Comment:
I've decided to go with less article-oriented class name so it could be used on other views without rising eyebrows :)